test/suites/auth.sh

```diff
@@ -143,6 +143,12 @@ EOF
   # Perform access check compatibility with project feature flags
   auth_project_features
 
+  # The OIDC identity should be able to delete themselves without any permissions.
```

Comment:
Will the tests for TLS fine grained come next?

Reply:
Yep they're in #14207 :)